src/main/java/teammates/common/datatransfer/FeedbackResponseCommentAttributes.java

```diff
@@ -81,6 +81,14 @@ public FeedbackResponseCommentAttributes(String courseId, String feedbackSession
         this.lastEditedAt = createdAt;
     }
     
+    public FeedbackResponseCommentAttributes(String courseId, String feedbackSessionName,
```

Comment:
It's specialized for the new constructor in `FeedbackResponseComment`.
An alternative would be to include those variables into the constructor for that instead of an `FeedbackResponseCommentAttributes`.